lib/pure/os.nim

```diff
@@ -1059,6 +1062,7 @@ template rawToFormalFileInfo(rawInfo, path, formalInfo): untyped =
     else:
       formalInfo.permissions = {fpUserExec..fpOthersRead}
 
+    formalInfo.isRegular = true
```

Comment:
This would not be necessary then.

Reply:
Sure, but `isFragile` isn't the opposite of `isRegular` - that would be `isSpecial` or `isIrregular`.
Searching through the manpages on my system, "regular file" and "special file" seem to be used in equal measure:
```
> man --where --regex --global-apropos "(file[^.]+regular)|(regular[^.]+file)" | wc -l
160
[13:25:29] clay@XPS /c/Users/Clay Sweetser
> man --where --regex --global-apropos "(file[^.]+special)|(special[^.]+file)" | wc -l
158
```

I think, ideally, there would be a field for one term, and a template for the other, so that users are free to use either `isSpecial` or `isRegular`.